treeherder/model/error_summary.py

```diff
@@ -65,9 +94,31 @@ def bug_suggestions_line(err, term_cache=None):
     if term_cache is None:
         term_cache = {}
 
+    # store "search_terms: count"
+    if logdate is None:
+        # use today
+        today = str(datetime.datetime.now().date())
+    else:
+        today = str(logdate.date())
+    if today not in line_cache.keys():
+        line_cache[today] = {}
+
     # remove the mozharness prefix
     clean_line = get_cleaned_line(err.line)
 
+    # remove floating point numbers from the summary as they are often variable
+    cache_clean_line = re.sub(r' [0-9]+\.[0-9]+ ', ' X ', clean_line)
+    cache_clean_line = re.sub(r' leaked [0-9]+ window(s)', ' leaked X window(s)', cache_clean_line)
```

Comment:
Is the number of leaked windows random?

Reply:
I see 1,2,3,4 in my 5 weeks of downloaded failure data.